tests/server/api/orderItems.test.js

```diff
@@ -35,17 +35,8 @@ afterAll(async () => {
 describe('/api/orderItems/', () => {
   let cookie;
 
-  describe('is shielded from logged out clients', () => {
-    test('sends status 401 to a logged out client', async () => {
-      try {
-        const res = await server.get('/api/orderItems/');
-        expect(res.status).toBe(401);
-      } catch (error) {
-        throw error;
-      }
-    });
-
-    test('lets logged in clients make requests', async () => {
+  describe('get cookie', () => {
+    test('lets clients make requests', async () => {
```

Comment:
Is there another test that can be put in the place of the one that was removed? @justin-cook-developer